platform/viewer/cypress/integration/common/OHIFCornerstoneToolbar.spec.js

```diff
@@ -0,0 +1,509 @@
+describe('OHIF Cornerstone Toolbar', () => {
+  
+  before(() => {
+      cy.openStudy("MISTER^MR");
+      cy.waitDicomImage();
+    });
+
+    it('checks if all primary buttons are being displayed', () => {
+      //Creating Aliases
+      cy.get('.toolbar-button:nth-child(2)').as('stackScrollBtn');
+      cy.get('.toolbar-button:nth-child(3)').as('zoomBtn');
+      cy.get('.toolbar-button:nth-child(4)').as('levelsBtn');
+      cy.get('.toolbar-button:nth-child(5)').as('panBtn');
+      cy.get('.toolbar-button:nth-child(6)').as('lengthBtn');
+      cy.get('.toolbar-button:nth-child(7)').as('annotateBtn');
+      cy.get('.toolbar-button:nth-child(8)').as('angleBtn');
+      cy.get('.toolbar-button:nth-child(9)').as('resetBtn');
+      cy.get('.toolbar-button:nth-child(10)').as('cineBtn');
+      cy.get('.expandableToolMenu').as('moreBtn');
+      cy.get('.PluginSwitch > .toolbar-button').as('twodmprBtn');
+      cy.get('.btn-group > .toolbar-button').as('layoutBtn');
+
+      cy.get('@stackScrollBtn').should('be.visible').contains('Stack Scroll');
+      cy.get('@zoomBtn').should('be.visible').contains('Zoom');
+      cy.get('@levelsBtn').should('be.visible').contains('Levels');
+      cy.get('@panBtn').should('be.visible').contains('Pan');
+      cy.get('@lengthBtn').should('be.visible').contains('Length');
+      cy.get('@annotateBtn').should('be.visible').contains('Annotate');
+      cy.get('@angleBtn').should('be.visible').contains('Angle');
+      cy.get('@resetBtn').should('be.visible').contains('Reset');
+      cy.get('@cineBtn').should('be.visible').contains('CINE');
+      cy.get('@moreBtn').should('be.visible').contains('More');
+      cy.get('@twodmprBtn').should('be.visible').contains('2D MPR');
+      cy.get('@layoutBtn').should('be.visible').contains('Layout');
+    });
+
+
+    it('checks if Stack Scroll tool will navigate across all series in the viewport', () => {
+      cy.waitDicomImage();
+      //Click on button and vefiry if icon is active on toolbar
+      cy.get('.toolbar-button:nth-child(2)').as('stackScrollBtn')
+        .click()
+        .then(($stackScrollBtn) =>{
+          cy.wrap($stackScrollBtn)
+            .should('have.class', 'active')
+        })
+
+      //drags the mouse inside the viewport to be able to interact with series
+      cy.get('.viewport-element')
+        .trigger('mousedown', 'top', { which: 1 })
+        .trigger('mousemove', 'center', { which: 1 })
+        .trigger('mouseup');
+
+      const overlaySeriesInformation = 'div.ViewportOverlay > div.bottom-left.overlay-element > div';
+      const expectedText = 'Ser: 1Img: 14 14/26256 x 256Loc: 0.00 mm Thick: 5.00 mm';   
+      cy.get(overlaySeriesInformation)
+        .should('have.text', expectedText);
+
+      cy.resetViewport();
```

Comment:
Similar to your `before(() => {});` hook above, you should have an `afterEach` you can leverage for this

Reply:
Thanks for your help on this one! :D